CHANGELOG.rst

```diff
@@ -13,7 +28,7 @@ Release Summary
 
 This is the minor release of the ``community.rabbitmq`` collection.
 This changelog contains all changes to the modules and plugins in this collection
-that have been made after release 1.1.0.
+that have been made after the 1.1.0 release.
```

Comment:
Note that this is here in this release commit because I changed this file for the `1.2.0` release after `antsibull-changelog` had been run, not realising that it would get re-written entirely the next time... sorry about that!